packages/prisma/src/prisma/schema.prisma

```diff
@@ -199,25 +261,117 @@ model Question {
   updatedAt DateTime @updatedAt
 }
 
-model Tag {
-  id   Int    @id @default(autoincrement())
-  name String
+// TODO: elements are only supported for the new PRACTICE_QUIZ type in the first iteration. add the other types here when working on them.
+enum ElementInstanceType {
+  PRACTICE_QUIZ
+}
 
-  order Int @default(0)
+model ElementInstance {
+  id Int @id @default(autoincrement())
 
   originalId String? @unique
 
-  questions Question[]
+  type        ElementInstanceType
+  elementType ElementType
+  order       Int
 
-  owner   User   @relation(fields: [ownerId], references: [id], onDelete: Cascade, onUpdate: Cascade)
-  ownerId String @db.Uuid
+  // TODO: moved to options Json
+  // resetTimeDays    Int?
+  // pointsMultiplier Int                      @default(1)
+  // TODO: moved to results Json
+  // participants    Int                      @default(0)
+
+  /// [PrismaElementInstanceOptions]
+  options Json // contains element type specific settings (resetTimeDays for LE instances, pointsMultiplier where relevant)
```

Comment:
```suggestion
  activityOptions Json // contains element type specific settings (resetTimeDays for LE instances, pointsMultiplier where relevant)
```

Reply:
activityOptions probably does not make sense either, as e.g., the points multiplier could be moved here and that is the combination of the activity and element multiplier. will leave it at options for now.. maybe "derivedOptions" or something could be used..